### PR TITLE
Khalil Alhaydr - London10 - JavaScript-Core2-Coursework-Week3

### PR DESCRIPTION
**Pull request by Khlil1313**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

**Comment by SallyMcGrath** [comment]
Hi @Khlil1313 . Please update your PR title to match the requested format. Thanks!

(Look at the list of PRs in this repo to see how everyone else is doing it.)

**Comment by Khlil1313** [comment]
Hi Sally,
When I started the JavaScript core 2 coursework week3, I forgot to fork the
repo first and then I had an issue when I was trying to push the changes to
main. Ibrhim from CYF helped me to do another forked repo and then push the
changes to the new forked repo. So, shall I delete this PR?
Thanks

File: 1-exercises/A-reading-list/script.js [hunk 1]
```diff
@@ -1,5 +1,32 @@
 function readingList(books) {
-  // Write your code here...
+  let ulElement = document.createElement("ul")
+  content.appendChild(ulElement)
```
Comment (kitko112) [inline 1-exercises/A-reading-list/script.js:3]: What is the `content` here? Do you think  document.querySelector  for content is necessary?

File: 1-exercises/A-reading-list/script.js [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+  for (let book of books) { 
+    let pElement = document.createElement("p")
+    
+    pElement.innerText = `${book.title} by ${book.author} `
+
+    let imgElement = document.createElement("img")
+    imgElement.setAttribute("src", book.bookCoverImage)
```
Comment (kitko112) [inline 1-exercises/A-reading-list/script.js:10]: Nice way of adding image by setAttribute, fyi here is another way that also works `imgElement.src`.

File: 1-exercises/B-highlight-words/script.js [hunk 3]
```diff
@@ -1,10 +1,46 @@
...
-
 const colours = ["yellow", "green", "blue", "none"];
 
+function changeColour (colour) {
+  let colourBox = document.querySelector("#colourSelect");
+  if (colourBox.value === "none") {
+    this.style.backgroundColor = null
```
Comment (kitko112) [inline 1-exercises/B-highlight-words/script.js:9]: Nice work. fyi backgroundColor can be 'none' too, it still works if you remove the if (colourBox.value === "none") checking

**Pull request closed without merging**